torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -384,6 +384,20 @@ class InsertObserversHelper {
   // These are the IR patterns we match to skip inserting observers.
   // They are compiled once on construction and used repeatedly within
   // the pass.
+  const PatternInfo conv1d_functional_relu = PatternInfo::parse_from_str(R"(
```

Comment:
let's use naming convention suggested by @z-a-f : `nn_conv1d_f_relu`, similar for others

Reply:
Sure